src/fetch/orderbook.py

```diff
@@ -49,9 +49,6 @@ def _pg_engine(db_env: OrderbookEnv) -> Engine:
     def _read_query_for_env(
         cls, query: str, env: OrderbookEnv, data_types: Optional[dict[str, str]] = None
     ) -> DataFrame:
-        # It seems there is a bug in mypy on the dtype field (with error [arg-type]).
-        # They expect types that should align with what we pass.
-        # More context at: https://github.com/cowprotocol/dune-sync/issues/24
         return pd.read_sql_query(query, con=cls._pg_engine(env), dtype=data_types)
```

Comment:
This was supposed to be removed in #46 but forgotten. Slipping it in here. Also related to #24 (which is now closed)